os/hal/include/rng.h

```diff
@@ -0,0 +1,136 @@
+/*
+    RNG for ChibiOS - Copyright (C) 2016 Stephane D'Alu
```

Comment:
If you don't have too strong opinions on the copyright here I would suggest attributing this file to Giovanni Di Sirio because this looks like something that could be worked on to be upstream to the main ChibiOS repo and if that is to happen I think it would be very unlikely that Giovanni would accept code attributed to anyone else. But I'm not a lawyer so I'm not sure if this is something right to do...

@barthess What do you think about this?